ops/graph-deploy.js

```diff
@@ -15,22 +15,21 @@ async function deploy (opts = {}) {
   let msg
 
   /* create the subgraph */
-  result = await runGraphCli([
-    'create',
-    '--access-token ""',
-    '--node ' + graphNode,
-    opts.subgraphName
-  ], cwd)
-  msg = result[1] + result[2]
-  // if (result[0] === 1) {
-  //   console.error(`Create failed! ${msg}`)
-  // }
-  if (msg.toLowerCase().indexOf('error') > 0) {
-    if (msg.match(/subgraph already exists/)) {
-      // the subgraph was already created before -we're ok
-      console.log('subgraph already exists - deploying a new version')
-    } else {
-      console.error(`Create failed! ${msg}`)
+  if (graphNode !== 'https://api.thegraph.com/deploy/') {
```

Comment:
what is the 'else' of this condition ?

Reply:
There should be no else.

Reply:
why this is needed ?

Reply:
Because you can't create subgraph which is on The Graph's hosted services, so only if it's not that you should create it.

Reply:
oh, ok ..need to be able to do that.